tests/e2e/e2e_groups_test.go

```diff
@@ -74,26 +74,26 @@ func (s *IntegrationTestSuite) GroupsSendMsgTest() {
 	s.prepareGroupFiles(ctx, s.chainA, adminAddr.String(), aliceAddr, bobAddr, charlieAddr)
 
 	s.T().Logf("Creating Group")
-	s.execCreateGroup(s.chainA, 0, chainAAPIEndpoint, adminAddr.String(), "Cosmos Hub Group", fmt.Sprintf("/root/.gaia/config/%s", originalMembersFilename), fees.String())
```

Comment:
/root/.gaia/config/%s should be a const

Reply:
I think actually this might be the problem, I'm going to confirm what the data dir actually is

Reply:
I slacked you earlier ... I've committed to this branch already the fix
swap /root/.gaia/config/%s  for /home/nonroot/.gaia/config/%s

Reply:
👍